### PR TITLE
[stable10] Test files:scan for specific users (negative validation)

### PR DESCRIPTION
**Pull request by skshetry**
Backport for #33747

### CODE REVIEW

**Review by phil-davis — APPROVED** [review]

File: tests/TestHelpers/UploadHelper.php [hunk 1]
```diff
@@ -114,7 +114,16 @@ public static function upload(
 				$filename = $v2ChunksDestination . '/' . (string)($index);
 				$davRequestType = "uploads";
 			}
-			$result = WebDavHelper::makeDavRequest(
+			$result = WebDavHelper::makeDavRequest Total 5 min Monthly Balance -22 h 26 min
```
Comment (phil-davis) [inline tests/TestHelpers/UploadHelper.php]: @paurakhsharma something went wrong here

**Review by phil-davis — CHANGES_REQUESTED** [review]
Rebase has accidental code changes

File: tests/TestHelpers/UploadHelper.php (hunk 1 above)
Comment (paurakhsharma) [reply tests/TestHelpers/UploadHelper.php]: @skshetry Pasted his tmetric stats here 😄

**Comment by skshetry** [comment]
@ownclouders rebase

**Comment by ownclouders** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being rebased automatically. Please **DO NOT** push while rebase is in progress or your changes would be lost permanently :warning:

**Comment by ownclouders** [comment]
Automated rebase with [GitMate.io](https://gitmate.io) was successful! :tada:

**Review by phil-davis — APPROVED** [review]
Looks good now

**Pull request merged**